code/extensions/ScheduledExecutionExtension.php

```diff
@@ -18,16 +20,21 @@ class ScheduledExecutionExtension extends DataExtension {
 		'ExecuteFree'			=> 'Varchar',
```

Comment:
Good to know there isn't a fix definition. I used to do un-aligned definition and switched later on because I felt its better to read. But that might be just me.

I had a quick look at "Clean Code" and going to give it a shot. It sounds like a good book to read.